plugins/asus-hid/asus-hid.quirk

```diff
@@ -21,9 +21,11 @@ Flags = use-runtime-version
 Plugin = asus_hid
 Flags = is-bootloader,can-verify-image
 FirmwareSizeMax = 0x40000
+GType = FuAsusHidDevice
```

Comment:
I think if you use fu_plugin_set_device_gtype_default() you might be able to drop this.

Reply:
fixed, thanks!